app/views/vacancies/_show.html+phone.haml

```diff
@@ -0,0 +1,123 @@
+- content_for :page_title_prefix do
+  #{@vacancy.job_title} — #{@vacancy.school_name}
+
+.vacancy.grid-row
+  - if @vacancy.expired?
+    .column-full
+      .banner-warning
+        = t('jobs.expired')
+
+  .column-one-third
+    %h1.heading-large
+      = @vacancy.job_title
+      %span.heading-secondary
+        = @vacancy.location
+
+    %h2.heading-medium
+      = t('schools.about')
+      = @vacancy.school.name
+    %p= @vacancy.school.description
+
+    %dl.school--details
```

Comment:
Do you think we should move the school details to a partial? It looks like it uses the same content as the normal `_show.html.haml`

Reply:
👍